Stratis.BitcoinD/Program.cs

```diff
@@ -21,46 +21,68 @@ public static void Main(string[] args)
 			Logs.Configure(new LoggerFactory().AddConsole(LogLevel.Trace, false));
 			NodeSettings nodeSettings = NodeSettings.FromArguments(args);
 
-			var node = (FullNode) new FullNodeBuilder()
-				.UseNodeSettings(nodeSettings)
-				.UseBlockStore()
-				.UseMempool()
-				.Build();
-
-			// == shout down thread ==
-			new Thread(() =>
-			{
-				Console.WriteLine("Press one key to stop");
-				Console.ReadLine();
-				node.Dispose();
-			})
+			try	
 			{
-				IsBackground = true //so the process terminate
-			}.Start();
+				// Verify folders are accessible by attempting to create sub directories,
+				// and removing them after the test.
+				var dataFolder = new DataFolder(nodeSettings);
+				foreach(var path in new []{ dataFolder.BlockPath, dataFolder.ChainPath, dataFolder.CoinViewPath }) {
```

Comment:
Sorry, things came up today, and I was very busy, but hopefully will have this done tomorrow.